Cargo.toml

```diff
@@ -28,7 +28,7 @@ members = [
 ]
 
 [profile.dev]
-opt-level = 2
+opt-level = 0
```

Comment:
nooooooo

Reply:
Yes, yes, yes! — as the youth would say.

After tests, it seems that generated wasm size difference (which was the reason for having opt level 2 here) is ~10% and does not block this change.
A nice side-effect is that it makes CI twice as fast.